samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -78,18 +82,22 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void cleanUp() throws Exception {
+  public void cleanUp()  {
     // clean up the temp files created
-    FileUtils.deleteDirectory(taskStoreDir);
+    try {
+      FileUtils.deleteDirectory(taskStoreDir);
+    } catch (IOException e) {
+      // Happens when task store can't be deleted after test finishes.
+      Assert.fail();
```

Comment:
Assert.fail doesn't take exception. 

It takes in message as argument. Added e.getMessage() to it. 

`e.printStackTrace()` might not be useful, it depends if gradle setting redirects stdout/LOG messages in test to build output.